src/router.ts

```diff
@@ -91,21 +89,21 @@ export class Router extends EventEmitter {
         }
 
         try {
-          this.verbose(`Running found HTTP handler.`);
+          this.log.verbose(`Running found HTTP handler.`);
```

Comment:
I'd actually prefer to use `trace` on these instead of a `verbose` -- can we drop that method on `Logger` and move these to `trace`s?

Reply:
done!